### DISCUSSION TITLE
Missing youth KSADS y4 data?

### DISCUSSION THREAD

**Discussion by gabriellaatencio** [now-i-know-my-abcd/docs#72] [Where in the World is…? (Help)]
Hello! I'm interested in looking at the youth symptom-level KSADS year 4 data in 5.1 but it does not look like it's there. I am wondering if anybody is running into the same issue, or may know why? Thanks!

**Comment by ebrahimebrahim** [comment]
From [the 5.1 release notes here](https://wiki.abcdstudy.org/release-notes/start-page.html#sample):

> ...the 42-month and 4-year follow-up events were still ongoing when the data for this release was frozen, so data are included for a subset of participants for these timepoints.
>
> The following table lists the number of included participants per event:
> 
> Event | n
> -- | --
> baseline | 11868
> 6-month follow-up | 11389
> 1-year follow-up | 11220
> 18-month follow-up | 11083
> 2-year follow-up | 10973
> 30-month follow-up | 10228
> 3-year follow-up | 10336
> 42-month follow-up | 8449
> 4-year follow-up | 4754
>

**Reply by gabriellaatencio** [reply]
Hello, thank you for your response! Although that is correct, the entire dataset is entirely empty such that there are not even subject numbers or event names. 

I am looking at the mh_y_ksads_ed.csv and mh_y_ksads_ss.csv CSVs. I would be very appreciative if someone else could possibly please check their copy and confirm that the problem is not only on my end, where I didn't just get a bad file or something.

**Reply by ebrahimebrahim** [reply]
Oh I misunderstood sorry. My copies of mh_y_ksads_ed.csv and mh_y_ksads_ss.csv are nonempty files with 10801 and 48768 lines respectively. (It's possible I had downloaded those from the 5.0 and not the 5.1 release however.)

**Reply by gabriellaatencio** [reply]
Due to known issues with the KSADS in 5.0, we need to use 5.1. Thank you so much for checking!

**Reply by emilygiddens** [reply]
Hi there, 

I am having a similar issue. My parental KSADS eating disorder dataset (mh_p_ksads_ed.csv) from the 5.1 release do not contain 4 yr follow-up data. I can see Baseline and 2 year follow-up data. Just wondering if there was no 4 year follow-up data from ED module data released in the 5.1 release? If it's available, I'd appreciate someone pointing me in the direction of where it could be. :)

Thank you!

**Reply by laikevin128** [reply]
You can check the mh_p_ksads_ss.csv file for fuller data.
